markdownPreview/index.ts

```diff
@@ -1,10 +1,18 @@
 import mermaid, { MermaidConfig } from 'mermaid';
 import { renderMermaidBlocksInElement } from './mermaid';
+import * as logos from '@iconify-json/logos';
 
-function init() {
+function init() { 
     const configSpan = document.getElementById('markdown-mermaid');
     const darkModeTheme = configSpan?.dataset.darkModeTheme;
     const lightModeTheme = configSpan?.dataset.lightModeTheme;
+  
+    mermaid.registerIconPacks([
```

Comment:
This should likely happen in shared code, see `extendMarkdownItWithMermaid`. That way it works both in notebooks and the markdown previews

Reply:
I have created `registerIconPacks` in `src/mermaid.ts `.